platform/jobs/edxTravisCounts.groovy

```diff
@@ -0,0 +1,57 @@
+package platform
+
+Map config = [:]
+Binding bindings = getBinding()
+config.putAll(bindings.getVariables())
+PrintStream out = config['out']
+
+// Travis Jobs
+
+// The travis-counts job
+Map prJob = [ name: 'travis-counts',
+              deprecated: false,
+              worker: 'coverage-worker',
+              branch: '*/master'
+              ]
+
+/* Iterate over the job configurations */
+secretMap.each { jobConfigs ->
+
+    Map jobConfig = jobConfigs.getValue()
+
+    /* Test secret contains all necessary keys for this job */
+    assert jobConfig.containsKey('open')
+    assert jobConfig.containsKey('jobName')
+    assert jobConfig.containsKey('url')
+    assert jobConfig.containsKey('subsetJob')
+
+    job(jobConfig['jobName']) {
+
+        logRotator {
+            daysToKeep(5)
+            numToKeep(201)
+        }
+        concurrentBuild(true)
+        label('coverage-worker')
+        environmentVariables {
+            env('SUBSET_JOB', jobConfig['subsetJob'])
+        }
+        scm {
```

Comment:
I was confused about why you had 'multijob', then I realized- the job on Jenkins is incorrect! There is only one git repo, so no need to use 'multijob/multiscm'. You can simply use git. See here for an example: https://github.com/edx/jenkins-job-dsl/blob/master/testeng/jobs/toggleSpigot.groovy#L65